torch/csrc/profiler/python/init.cpp

```diff
@@ -314,17 +413,24 @@ void initPythonBindings(PyObject* module) {
       "_set_cuda_sync_enabled_val",
       &torch::profiler::impl::set_cuda_sync_enabled_val);
 
-  py::class_<CapturedTraceback, std::shared_ptr<CapturedTraceback>>(
-      m, "CapturedTraceback");
+  TORCH_CHECK(PyType_Ready(&THPCapturedTracebackType) >= 0);
+  PyModule_AddObject(
+      m.ptr(), "CapturedTraceback", (PyObject*)&THPCapturedTracebackType);
   m.def(
       "gather_traceback",
       CapturedTraceback::gather,
       py::arg("python") = true,
       py::arg("script") = true,
       py::arg("cpp") = true);
-  m.def("symbolize_tracebacks", py_symbolize);
+  m.def("symbolize_tracebacks", [](py::list tbs) {
+    std::vector<CapturedTraceback*> tb_ptrs;
+    tb_ptrs.reserve(tbs.size());
+    for (py::handle tb : tbs) {
+      tb_ptrs.emplace_back(((THPCapturedTraceback*)tb.ptr())->data.get());
+    }
+    return py_symbolize(tb_ptrs);
```

Comment:
Having to do this conversion by hand is annoying, the type caster doesn't seem to be enough. Also, adding a type caster for CapturedTraceback* doesn't work